packages/core/src/modules/dids/methods/sov/__tests__/IndySdkSovDidRegistrar.test.ts

```diff
@@ -4,51 +4,58 @@ import type * as Indy from 'indy-sdk'
 
 import { getAgentConfig, getAgentContext, mockFunction, mockProperty } from '../../../../../../tests/helpers'
 import { SigningProviderRegistry } from '../../../../../crypto/signing-provider'
+import { ConsoleLogger } from '../../../../../logger'
 import { JsonTransformer } from '../../../../../utils/JsonTransformer'
 import { IndyWallet } from '../../../../../wallet/IndyWallet'
-import { IndyLedgerService } from '../../../../ledger/services/IndyLedgerService'
 import { IndyPoolService } from '../../../../ledger/services/IndyPoolService'
 import { DidDocumentRole } from '../../../domain/DidDocumentRole'
 import { DidRepository } from '../../../repository/DidRepository'
-import { SovDidRegistrar } from '../SovDidRegistrar'
+import { IndySdkSovDidRegistrar } from '../IndySdkSovDidRegistrar'
 
 jest.mock('../../../repository/DidRepository')
 const DidRepositoryMock = DidRepository as jest.Mock<DidRepository>
 
-jest.mock('../../../../ledger/services/IndyLedgerService')
-const IndyLedgerServiceMock = IndyLedgerService as jest.Mock<IndyLedgerService>
-
 jest.mock('../../../../ledger/services/IndyPoolService')
 const IndyPoolServiceMock = IndyPoolService as jest.Mock<IndyPoolService>
 const indyPoolServiceMock = new IndyPoolServiceMock()
 mockProperty(indyPoolServiceMock, 'ledgerWritePool', { config: { id: 'pool1', indyNamespace: 'pool1' } } as IndyPool)
+mockFunction(indyPoolServiceMock.getPoolForNamespace).mockReturnValue({
+  config: { id: 'pool1', indyNamespace: 'pool1' },
+} as IndyPool)
 
-const agentConfig = getAgentConfig('SovDidRegistrar')
+const agentConfig = getAgentConfig('IndySdkSovDidRegistrar')
 
 const createDidMock = jest.fn(async () => ['R1xKJw17sUoXhejEpugMYJ', 'E6D1m3eERqCueX4ZgMCY14B4NceAr6XP2HyVqt55gDhu'])
 
 const wallet = new IndyWallet(agentConfig.agentDependencies, agentConfig.logger, new SigningProviderRegistry([]))
 mockProperty(wallet, 'handle', 10)
 
+jest.mock('../../../../../logger/Logger')
+const LoggerMock = ConsoleLogger as jest.Mock<ConsoleLogger>
```

Comment:
We usually just take the logger from the agentConfig.logger instead of mocking it